pkg/config/autodiscover/csi_info.go

```diff
@@ -0,0 +1,37 @@
+package autodiscover
+
+import (
+	"fmt"
+	"strings"
+)
+
+// PodSetList holds the data from an `oc get deployment/statefulset -o json` command
+
+// PodSetResource defines deployment/statefulset resources
+
+func getpackageandorg(csi string) (packag, organization string) {
+	orgpack := fmt.Sprintf("./get-csi-info.sh %s", csi)
```

Comment:
I'm not sure about this. We're calling a script to get the information, but we have no guarantee about the script location (we could be running the test suite from a different directory, and maybe it's not included in the container).

Is there a better way to get that information from CSI drivers?

Reply:
I agree. Isn't it pretty easy to invoke utils.ExecuteCommand function a few times to do the same thing?